### ISSUE TITLE
Accessibility suggestion

### ISSUE THREAD

**Issue by nikosdrog** [baianat/hooper#250]
Hello there.
I have a suggestion about the accessibility . The navigation buttons does not have the aria-label.
When I use the hooper-prev and hooper-next slot then it will be good to be able to edit the button, not only the inside of the button.

**Comment by Gabriella-roze** [comment]
👆 Yep, the accessibility upgrade for the hoper navigation would be a great addition.
